src/test/scala/freeeventsourcing/ProcessDefinitionTests.scala

```diff
@@ -0,0 +1,27 @@
+package freeeventsourcing
+
+import freeeventsourcing.accountprocessing.Account.Command.BlockFunds
+import freeeventsourcing.accountprocessing.Account.Error.{ InsufficientFunds, NotOpen }
+import freeeventsourcing.accountprocessing.Transaction.Event.Created
+import freeeventsourcing.accountprocessing.{ Account, AccountProcessing, Transaction }
+import freeeventsourcing.syntax.ProcessSyntax
+import org.scalatest.{ FlatSpec, Matchers }
+
+class ProcessDefinitionTests extends FlatSpec with Matchers {
+  val syntax = new ProcessSyntax(AccountProcessing)
+  import syntax._
+
+  "ProcessDefinition " should " allow to define a process" in {
+    //TODO selector for all Open events...
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Take the required action to fix the issue indicated by this comment.](https://www.codacy.com/app/msiegenthaler/free-event-sourcing/file/2650784387/issues/source?bid=3298199&fileBranchId=3311368#l15)